src/screen/CreateAccount/MnemonicScreen.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import SafeView, {noTopEdges} from 'presentational/SafeView';
+import {View, StyleSheet} from 'react-native';
+import {Text, Input, Icon, Button} from '@ui-kitten/components';
+import {mnemonicGenerate} from '@polkadot/util-crypto';
+import globalStyles, {monofontFamily, standardPadding} from 'src/styles';
+import {NavigationProp} from '@react-navigation/native';
+import {AccountsStackParamList} from 'src/navigation/navigation';
+import FormLabel from 'presentational/FormLabel';
+import Padder from 'presentational/Padder';
+import {verifyMnemonicScreen} from 'src/navigation/routeKeys';
+
+export function MnemonicScreen({navigation}: {navigation: NavigationProp<AccountsStackParamList>}) {
+  const [mnemonic, setMnemonic] = React.useState('');
```

Comment:
I think you can also do [lazy state initialisation](https://reactjs.org/docs/hooks-reference.html#lazy-initial-state) instead of the effect.